src/nunit.uikit/Controls/TipWindow.cs

```diff
@@ -177,6 +177,13 @@ protected override void OnLoad(System.EventArgs e)
 
 			Size sizeNeeded = Size.Ceiling( g.MeasureString( tipText, Font, layoutArea ) );
 
+            // If the needed width is smaller than that of the original label,
```

Comment:
This whole file has tabs, you used spaces. not super important. For the purpose of consistency can this switch to tabs? :cry: